string  status

uint8 DIR_1 = 1
uint8 DIR_2 = 2
uint8 DIR_3 = 4
uint8 DIR_4 = 8

uint8 direction

uint8 POSE_N = 1
uint8 POSE_E = 2
uint8 POSE_S = 4
uint8 POSE_W = 8

uint8 pose_mask